dpnp/dpnp_iface_histograms.py

```diff
@@ -208,6 +209,98 @@ def _search_sorted_inclusive(a, v):
     )
 
 
+def digitize(x, bins, right=False):
+    """
+    Return the indices of the bins to which each value in input array belongs.
+
+    For full documentation refer to :obj:`numpy.digitize`.
+
+    Parameters
+    ----------
+    a : {dpnp.ndarray, usm_ndarray}
+        Input array to be binned.
+    bins : {dpnp.ndarray, usm_ndarray}
```

Comment:
Numpy also supports a scalar for `bins`. What do you think to add support of scalar here in dpnp?

Reply:
You mean for `x`?

Reply:
Numpy does not support `bins` as a scalar
```
a = numpy.array([1,2,3,4,5])
numpy.digitize(a,2)


>>> ValueError: object of too small depth for desired array
```